test/Optimizer/test149.js

```diff
@@ -0,0 +1,23 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+var obj0 = {};

```

Comment:
I found that
```js
function func2() {
  for (; Object.noprop;) {}
}
var ary = [1];
var uic8 = new Uint8ClampedArray();
let i = 0;
while (ary[uic8[1] >= 0 ? uic8[1] : 0]) {
  func2();
  if (i++ > 1000) {
    break;
  }
}
print("passed");
```
Also reproduces the issue with no rl flags needed

Reply:
Ah, thanks.